lpcompiler.pas

```diff
@@ -1411,7 +1421,7 @@ function TLapeCompiler.ParseMethodHeader(out Name: lpString; addToScope: Boolean
         LapeExceptionFmt(lpeCannotOverrideOperator, [op_name[op], ltyp.AsString, rtyp.AsString], Pos);
     end;
 
-    if isFunction then
+    if isFunction or ((Result.MethodDef = mdProperty) and (Peek() = tk_sym_Colon)) then
```

Comment:
`isFunction` should maybe be renamed to `ExpectResult` or something more fitting? Not that important, though.